packages/editor-ui/src/stores/n8nRoot.store.ts

```diff
@@ -48,6 +48,15 @@ export const useRootStore = defineStore(STORES.ROOT, {
 			return `${this.baseUrl}${this.restEndpoint}`;
 		},
 
+		getRestCloudApiContext(): IRestApiContext {
+			return {
+				baseUrl: this.baseUrl.includes('stage-')
```

Comment:
should be more specific... `app-app.n8n.cloud` in case someone uses `stage` in their app name